hack/generator/pkg/astmodel/struct_type.go

```diff
@@ -127,5 +133,20 @@ func (structType *StructType) RelatedDefinitions(ref PackageReference, namehint
 		}
 	}
 
+	for name, function := range structType.functions {
+		funcDef := &FunctionDefinition{
+			Function:       function,
+			DefinitionName: DefinitionName{ref, name, &namehint},
+			Receiver:       &StructReference{DefinitionName{ref, namehint, nil}, false}} // TODO: This false right?
```

Comment:
Does Go have named parameters? Anonymous boolean values are icky.

Reply:
I think I fixed this